nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/InstitutionPoints.java

```diff
@@ -15,6 +16,15 @@ public record InstitutionPoints(
     BigDecimal institutionPoints,
     List<CreatorAffiliationPoints> creatorAffiliationPoints) {
 
+  public InstitutionPoints(
+      URI institutionId,
+      BigDecimal institutionPoints,
+      List<CreatorAffiliationPoints> creatorAffiliationPoints) {
+    this.institutionId = institutionId;
+    this.institutionPoints = adjustScaleAndRoundingMode(institutionPoints);
```

Comment:
Is this utility method part this class' logic? If not it might be worth injecting it into a factory method to make it clear that this is really configuration.